packages/atlas/src/components/_nft/NftCard/NftCard.tsx

```diff
@@ -25,22 +36,54 @@ export const NftCard: React.FC<NftCardProps> = ({
   fullWidth,
   loading,
 }) => {
+  const reflectionCells = Array.from({ length: 100 })
   return (
     <Container fullWidth={fullWidth}>
-      <VideoThumbnail clickable={false} {...thumbnail} />
-      <Details>
-        {loading ? <SkeletonLoader width="70%" height={24} bottomSpace={24} /> : <Title variant="h400">{title}</Title>}
-        <Content>
-          <Members loading={loading} caption="Creator" members={creator} />
-          {supporters && !!supporters.length && (
-            <>
-              <Members caption="Supporters" members={supporters} />
-              <Separator />
-            </>
+      {reflectionCells.map((_, idx) => {
+        const row = Math.floor(idx / 10)
+        const column = idx % 10
+
+        const lp = 50 + (column * 10 - 50) / 1
+        const tp = 50 + (row * 10 - 50) / 1
```

Comment:
```suggestion
        const lp = column * 10
        const tp = row * 10
```
I just copied this chunk of code from somewhere and started to tweak and left this not refactored. Let's just simplify these calculations or even move to `ReflectionGridCell`